web/js/link/link.ui.js

```diff
@@ -1,12 +1,18 @@
-var wv = wv || {};
-wv.link = wv.link || {};
+import $ from 'jquery';
+import throttle from 'lodash/throttle';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Clipboard from 'clipboard';
+import Share, {GA as googleAnalytics} from 'worldview-components';
+import util from '../util/util';
+import wvui from '../ui/ui';
 
-wv.link.ui = wv.link.ui || function (models, config) {
+export function linkUi(models, config) {
```

Comment:
What do you think about keeping the name `ui`, importing this into the `link` parent, and attaching it to the object that exports?